src/NServiceBus.Transport.SQS/SnsPreparedMessage.cs

```diff
@@ -21,13 +21,15 @@ public string MessageId
         public string Body { get; set; }
         public string Destination { get; set; }
         public long Size { get; private set; }
+        public long PayloadPaddingInBytes { get; set; }
```

Comment:
It might be a departure of how we already treat this type but given this property is kind of mandatory I wonder if we should make this init only and have further iterations to also clean up the other properties in a dedicated PR.